src/layouts/BaseLayout/BaseLayout.jsx

```diff
@@ -73,25 +74,27 @@ function BaseLayout({ children }) {
 
   return (
     <>
-      {isFullExperience ? (
-        <>
-          {newHeader ? <Header /> : <OldHeader />}
-          <GlobalTopBanners />
-        </>
-      ) : (
-        <Suspense fallback={null}>
-          {showDefaultOrgSelector && <InstallationHelpBanner />}
-        </Suspense>
-      )}
       <Suspense fallback={<FullPageLoader />}>
-        <ErrorBoundary sentryScopes={[['layout', 'base']]}>
-          <NetworkErrorBoundary>
-            <main className="container mb-8 mt-2 flex grow flex-col gap-2 md:p-0">
-              <GlobalBanners />
-              <OnboardingOrChildren>{children}</OnboardingOrChildren>
-            </main>
-          </NetworkErrorBoundary>
-        </ErrorBoundary>
+        <RepoBreadcrumbProvider>
```

Comment:
We need access to the breadcrumb context inside of Header on line 82, so I've moved it up from RepoPage to here.